Client/Frontend/Fakespot/FakespotCoordinator.swift

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+protocol FakespotCoordinatorDelegate: AnyObject {
+    // Define any coordinator delegate methods here
+}
+
+class FakespotCoordinator: BaseCoordinator {
+    weak var parentCoordinator: FakespotCoordinatorDelegate?
+    weak var delegate: FakespotCoordinatorDelegate?
+
+    let fakespotViewController: FakespotViewController
+    let viewModel: FakespotViewModel
+
+    init(router: Router) {
+        viewModel = FakespotViewModel()
+        fakespotViewController = FakespotViewController(viewModel: viewModel)
+        super.init(router: router)
+    }
+
+    func start() {
```

Comment:
Unit tests 👀 ?

Reply:
That got resolved, but there's no tests?

Reply:
I would rather have a specific test file as well for FakeSpotCoordinator. When this coordinator gets extended it's much easier to add tests if there's already a file for it, even if for now there's only a couple of tests that overlaps

Reply:
Updated